bin/setup

```diff
@@ -13,12 +13,13 @@ FileUtils.chdir APP_ROOT do
   # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
   # Add necessary setup steps to this file.
 
-  puts '== Installing dependencies =='
+  puts '== Installing ruby dependencies =='
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
   # Install JavaScript dependencies
```

Comment:
The comment is no longer necessary as we are saying the same in the next line. I'll remove it :)